### DISCUSSION TITLE
problem with verification

### DISCUSSION THREAD

**Discussion by isaacwanas** [RosettaCommons/rosetta#272] [ROSIE] [answered]
iam a master student in Cologne university of applied sciences (TH Koeln)and the institutional email is already in Github, however i don have an access to ROSIE, can you please helpme to fix that.
thanks in advance

**Comment by lyskov** [comment]
Could you please post your academic email domain here (the part of the email after `@` sign) so i can add it? Thanks,

**Comment by lyskov** [comment, marked as answer]
Is that `smail.th-koeln.de`? If so this should be fixed now, if you still have issues logging-in please feel free to re-open this issue.
